index.html

```diff
@@ -5612,25 +5533,174 @@ <h3>Remote Document and Context Retrieval</h3>
+              having an <a data-cite="HTML/semantics.html#attr-link-type">type attribute</a>
+              of <code>application/ld+json</code> along with the value of the
+              <a data-link-for="LoadDocumentOptions">profile</a> option, if found.</li>
+            <li>If <var>source</var> is undefined and the <a data-link-for="LoadDocumentOptions">extractAllScripts</a> option is not present, or <code>false</code>,
+              set <var>source</var> to the <a data-cite="DOM#dom-node-textcontent">textContent</a>
+              of the first <a>JSON-LD script element</a> in <var>document</var>.
+              <p>If no such element is found,
+                or the located element is not a <a>JSON-LD script element</a>,
+                the <var>promise</var> is rejected with a <a>JsonLdError</a> whose code is set to <a data-link-for="JsonLdErrorCode">loading document failed</a>
+                and processing is terminated.</p></li>
+            <li>If <var>source</var> is defined and the,
+              set <var>document</var> to the result of the
+              <a href="#extract-script-content">Extract Script Content algorithm</a>,
+              using <var>source</var>, rejecting <var>promise</var>
+              with a <a>JsonLdError</a> whose code set from the result, if an error is detected
+              and processing is terminated.
+            </li>
+            <li>Otherwise, if <var>source</var> is undefined and the <a data-link-for="LoadDocumentOptions">extractAllScripts</a> option is not present, or <code>false</code>,
+              the <var>promise</var> is rejected with a <a>JsonLdError</a> whose code is set to <a data-link-for="JsonLdErrorCode">loading document failed</a>
+              and processing is terminated.</li>              
+            <li>Otherwise, if <var>source</var> is undefined and the <a data-link-for="LoadDocumentOptions">extractAllScripts</a> option is <code>true</code>,
+              set <var>document</var> to a new empty <a>array</a>.
+              For each <a>JSON-LD script element</a> in <var>input</var>:
+              <ol>
+                <li>Set <var>source</var> to its <a data-cite="DOM#dom-node-textcontent">textContent</a>.</li>
+                <li>Set <var>script content</var> to the result of the <a href="#extract-script-content">Extract Script Content algorithm</a>,
+                  using <var>source</var>, rejecting <var>promise</var>
+                  with a <a>JsonLdError</a> whose code set from the result, if an error is detected
+                  and processing is terminated.</li>
+                <li>If <var>script content</var> is an <a>array</a>, merge it to the end of <var>document</var>.</li>
+                <li>Otherwise, append <var>script content</var> to <var>document</var>.</li>
+              </ol>
+            </li>
+          </ol>
+        </li>
+        <li>Otherwise, if the retrieved document's <a>Content-Type</a> is neither
```

Comment:
Same remark as above, this is an "else", not an "else if".
Suggested change (1/3)
```suggestion
        <li>Otherwise, the retrieved document's <a>Content-Type</a> is neither
          <code>text/html</code>,
```